templates/al2023/runtime/rootfs/etc/systemd/system/kubelet.service

```diff
@@ -6,6 +6,7 @@ Requires=containerd.service
 
 [Service]
 Slice=runtime.slice
+EnvironmentFile=/etc/eks/kubelet-environment
```

Comment:
Can we make this path /etc/eks/kubelet/environment?

Reply:
done 👍